Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -826,7 +824,7 @@ require({
             const html = defined(htmlFile) ? htmlFile.content : defaultHtml; // Use the default html for old gists
             applyLoadedDemo(code, html);
           })
-          .otherwise(function (error) {
+          .catch(function (error) {
```

Comment:
I doubt this functionality is ever used (unless we have some die hard Cesium folks still hanging on to Sandcastle gists) but since dojoscript is going to return a dojo when, `otherwise` is probably the right thing to have here. The other option is to wrap the `dojoscript.get` call in a `Promise.resolve` so it turns a thenable into a real promis.